Block/FormatterBlockService.php

```diff
@@ -23,7 +23,7 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class FormatterBlockService extends BaseBlockService
+class FormatterBlockService extends AbstractBlockService
```

Comment:
https://github.com/sonata-project/SonataBlockBundle/blob/3.x/Block/BaseBlockService.php

This is funny, the deprecate message is telling one service, but the class is actually extending another, maybe this change was wrong, will check! Thanks @TheCelavi can you provide a PR? Also we should change deprecation message, because it is missleading...